inc/namespace.php

```diff
@@ -22,6 +22,16 @@
 function bootstrap() {
 	add_action( 'plugins_loaded', __NAMESPACE__ . '\\load_elasticpress' );
 	add_filter( 'altis_healthchecks', __NAMESPACE__ . '\\add_elasticsearch_healthcheck' );
+
+	// Load debug bar for ElasticPress if Query Monitor is enabled in the config.
+	if ( get_config()['modules']['dev-tools']['query-monitor'] ?? false ) {
+
+		// Enable debugging for Elastic Press Debug Bar to display query logs.
+		if ( ! defined( 'WP_EP_DEBUG' ) || ! WP_EP_DEBUG ) {
```

Comment:
This could cause a warning / error. If a constant is defined already you can't define it again as that would defeat the purpose of constants :)